libs/labelbox/src/labelbox/schema/member.py

```diff
@@ -0,0 +1,433 @@
+        for project_membership in result["projectMemberships"]:
+            project_membership["role"] = Role(
+                self.client,
+                field_values={
+                    "id": project_membership["role"]["id"],
+                    "name": project_membership["role"]["name"],
+                },
+            )
+            project_membership["projectId"] = project_membership["project"][
+                "id"
+            ]
+
+            project_memberships.add(ProjectMembership(**project_membership))
+
+        return project_memberships
+
+    def get_members(
+        self,
+        search: str = "",
+        roles: Optional[List[Role]] = None,
+        group_ids: Optional[List[str]] = None,
+    ) -> Iterator["Member"]:
+        """
+        Gets all members in Labelbox.
+
+        Args:
+            search (str): Email of user you are looking to receive.
+            roles (Optional[list[Role]]): Role of the users you are wanting to receive.
+            group_ids (Optional[list[str]]): List of user group ids.
+
+        Returns:
+            Iterator[UserGroup]: An iterator over the user groups.
+        """
+        query = """
+            query GetOrganizationMembersPyApi(
+                $first: PageSize!
+                $skip: Int
+                $search: String
+                $roleIds: [String!]
+                $groupIds: [String!]
+                $complexFilters: [ComplexFilter!]
+                ) {
+                    users(
+                        where: {
+                        email_contains: $search
+                        organizationRoleId_in: $roleIds
+                        groupId_in: $groupIds
+                        complexFilters: $complexFilters
+                        }
+                        first: $first
+                        skip: $skip
+                    ) {
+                        id
+                        updatedAt
+                        createdAt
+                        name
+                        nickname
+                        email
+                        defaultRole {
+                            id
+                            name
+                        }
+                        picture
+                        isViewer
+                        isExternalUser
+                        canAccessAllProjects
+                        accessibleProjectCount
+                        userGroups {
+                            nodes {
+                                id
+                            }
+                        }
+                    }
+                }
+            """
+        previous_batch = 0
+        batch_size = 100
+        while True:
+            params = {
+                "first": batch_size,
+                "skip": previous_batch,
+                "search": search,
+                "roleIds": [role.uid for role in roles] if roles else None,
+                "groupIds": group_ids,
+                "complexFilters": None,
```

Comment:
These complexFilters are associated with Alignerr and can be added to support getting members based on filters